### PR TITLE
Target Blank for external link

### PR DESCRIPTION
**Pull request by designbybeck**
The icon on this looks to have an arrow showing an external link, but it loads in the same window. Added target="_blank"

### CODE REVIEW

File: apps/theming/templates/settings-admin.php [hunk 1]
```diff
@@ -136,7 +136,7 @@
 	<div class="theming-hints">
 		<?php if (!$_['canThemeIcons']) { ?>
 			<p class="info">
-				<a href="<?php p($_['iconDocs']); ?>">
+				<a href="<?php p($_['iconDocs']); ?> target="_blank">
```
Comment (nickvergessen) [inline apps/theming/templates/settings-admin.php:139]:
```suggestion
				<a href="<?php p($_['iconDocs']); ?> target="_blank" rel="noreferrer noopener">
```

Comment (skjnldsv) [reply apps/theming/templates/settings-admin.php:139]: @designbybeck can you address this? :)

**Review by skjnldsv — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Review by designbybeck — COMMENTED** [review]
This seems like a better solution. Thank you.

**Review by Pytal — APPROVED** [review]

**Comment by Pytal** [comment]
@designbybeck we'll need a signoff from you as well :)
```bash
git rebase HEAD~2 --signoff
git push --force-with-lease origin patch-1
```

**Comment by designbybeck** [comment]
Sorry all, I'm fairly new to all this git stuff. I'm still learning. I need
to check out the master/main branch? and then run that Pytal?
Thank you for helping me learn to contribute to NextCloud.

**Comment by Pytal** [comment]
@designbybeck no worries, thanks so much for contributing!

You may stay on the `patch-1` branch under your `designbybeck/server` fork repo and run the commands above :)

If you created this branch through GitHub, after cloning to your machine you may need to `git switch patch-1` to switch to this branch before running the commands 👆

**Comment by szaimen** [comment]
@designbybeck any update here?

**Comment by designbybeck** [comment]
Sorry All... again... I'm REALLY a newbie! I tried the code but got an error.

❯ git rebase HEAD~2 --signoff
First, rewinding head to replay your work on top of it...
Applying: Scope composer tools with the bin plugin
Applying: Output exception in cron
❯ git push --force-with-lease origin patch-1
error: src refspec patch-1 does not match any
error: failed to push some refs to 'https://github.com/nextcloud/server.git'

**Comment by Pytal** [comment]
@designbybeck no worries :) you'll need to run the commands in your forked version of the repo aka in the folder cloned with `git clone https://github.com/designbybeck/server.git`

**Comment by designbybeck** [comment]
OK I had to setup the personal token, but I think I finally got it. 
Enumerating objects: 17, done.
Counting objects: 100% (17/17), done.
Delta compression using up to 16 threads
Compressing objects: 100% (10/10), done.
Writing objects: 100% (12/12), 1.23 KiB | 1.23 MiB/s, done.
Total 12 (delta 8), reused 2 (delta 0)
remote: Resolving deltas: 100% (8/8), completed with 4 local objects.
....
 + f130c3ea5e...fc5e535111 patch-1 -> patch-1 (forced update)

**Pull request merged**